### DISCUSSION TITLE
where the model files stored in my computer?

### DISCUSSION THREAD

**Discussion by margo2130** [openai/whisper#2467] [Q&A] [answered]
hi, i downloaded the large model and i want to remove it. where is the model file in my computer? thanks.

**Comment by ryanheise** [comment]
See https://github.com/openai/whisper/discussions/1774

**Comment by ixn3rd3mxn** [comment, marked as answer]
in my pc me : C:\ Users\ Administrator\ .cache\ whisper
![image](https://github.com/user-attachments/assets/53bb2088-abf6-4f79-9d36-ecc8351e56e6)
